packages/atomic/src/components/common/interface/interface-common.tsx

```diff
@@ -26,6 +26,7 @@ export interface BaseAtomicInterface<EngineType extends AnyEngineType>
   host: HTMLStencilElement;
   bindings: AnyBindings;
   error?: Error;
+  compatibilityJson: 'v1' | 'v2' | 'v3' | 'v4';
```

Comment:
Why is v1 and v2 here ? Should this be `i18nCompatibilityJSONVersion` ?

Reply:
Whoops, could've sworn I removed 'v1' | 'v2' at one point, thanks for pointing this out! 

Also I agree, that name is more clear. I'll use that instead.